packages/common/src/pivotTable/pivotQueryResults.ts

```diff
@@ -41,19 +38,17 @@ type RecursiveRecord<T = unknown> = {
     [key: string]: RecursiveRecord<T> | T;
 };
 
-const isRecursiveRecord = (value: unknown): value is RecursiveRecord => {
-    return typeof value === 'object' && value !== null;
-};
+const isRecursiveRecord = (value: unknown): value is RecursiveRecord =>
```

Comment:
I don't know about you but I find this rule quite annoying

Reply:
I did'nt change anything on this functions, they were simply copied over. I prefer not to change the behaviour at the moment, unless you have a better suggestion

Reply:
all good! it's good to know that it's just not me 😊